src/main/java/step2/domain/LottoOrder.java

```diff
@@ -0,0 +1,20 @@
+package step2.domain;
+
+/**
+ * 로또 구매 요청
+ */
+public class LottoOrder {
+    // 로또 구매 단가 - 실제 로또 가격은 LottoList에서
+    private final Integer totalPrice;
+    private final Lottos lottos;
+
+    public LottoOrder(int totalPrice) {
+        this.totalPrice = totalPrice;
+        int count = totalPrice / Lotto.getLottoPrice();
+        this.lottos = new Lottos(count);
```

Comment:
```suggestion
        this.lottos = Lottos.createLottos(totalPrice)
```

반영을 위한 코멘트는 아닙니다.
한번 고민해보시면 좋을것 같습니다.

Reply:
좋은 부분 같습니다.